101-multiplevm-existing-vnet-storage/README.md

```diff
@@ -0,0 +1,28 @@
+# Create Multiple Virtual Machines(Windows/Ubuntu) with 2 NIC cards connecting to exisitng VNET Subnet's.
```

Comment:
isn't this duplicate with 101-deploy-multiple-vm-existing-vnet-storage? please remove